terragrunt/modules/acm-certificate/variables.tf

```diff
@@ -2,3 +2,8 @@ variable "domains" {
   description = "List of domain names included in the certificate"
   type        = list(string)
 }
+
+variable "legacy" {
```

Comment:
I'm not sure this is the right variable. It might be more straight-forward to make the variable more directly about what it's used for.

Reply:
My reasoning was that should only be set to `true` in the legacy account. 😅

Reply:
I get it - makes the code harder to understand IMO, but it's also not a big deal.

Reply:
I'm happy to improve this still. Just wanted to get this merged first to unblock more pull requests. 😅 Although I have to admit that I'm not sure how this can be made significantly better. More documentation maybe?